src/txmempool.h

```diff
@@ -138,6 +363,29 @@ class CTxMemPool
     void ApplyDeltas(const uint256 hash, double &dPriorityDelta, CAmount &nFeeDelta);
     void ClearPrioritisation(const uint256 hash);
 
+public:
+    void RemoveStaged(std::set<uint256>& stage);
+
+    /** When adding transactions from a disconnected block back to the mempool,
+     *  new mempool entries may have children in the mempool (which is generally
+     *  not the case when otherwise adding transactions).
+     *  UpdateTransactionsFromBlock will find child transactions and update the
```

Comment:
Would prefer if comments like this are written as "Name()" rather than just "Name" to make it clear what's a variable and what's a function.